src/script/components/calling/GroupVideoGrid.tsx

```diff
@@ -0,0 +1,177 @@
+/*
+ * Wire
+ * Copyright (C) 2020 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React, {useState, CSSProperties} from 'react';
+import {registerReactComponent} from 'Util/ComponentUtil';
+import SVGProvider from '../../auth/util/SVGProvider';
+import type {Grid} from '../../calling/videoGridHandler';
+import type {Participant} from '../../calling/Participant';
+import Video from './Video';
+import {t} from '../../util/LocalizerUtil';
+
+export interface GroupVideoGripProps {
+  grid: Grid;
+  minimized?: boolean;
+  muted?: boolean;
+  selfParticipant: Participant;
+}
+
+interface RowsAndColumns {
+  '--columns': number;
+  '--rows': number;
+}
+
+const calculateRowsAndColumns = (totalCount: number): RowsAndColumns => {
+  const columns = Math.ceil(Math.sqrt(totalCount));
+  const rows = Math.ceil(totalCount / columns);
+  return {'--columns': columns, '--rows': rows};
+};
+
+const GroupVideoGrid: React.FunctionComponent<GroupVideoGripProps> = ({
+  minimized,
```

Comment:
Does it make sense to make this `minimized = false`?

Reply:
https://github.com/wireapp/wire-webapp/pull/10025/commits/646a1dccf19bdba34f3f7e883bfa4ae8e248e17b